### PR TITLE
DOC-251-Nested-Cluster

### PR DESCRIPTION
**Pull request by Siboney1**
Nested Cluster and Quickstart page

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/854#" title="Author self-approved">Siboney1</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @Siboney1. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by Siboney1** [comment]
Includes Nested Cluster and Quickstart

**Review by TylerGillson — COMMENTED** [review]
Couple of minor nits... looks great overall though!

File: content/docs/04-clusters/4-nested-clusters.md
Comment (tiwarisumit3) [inline content/docs/04-clusters/4-nested-clusters.md:19]: nit pick: share selected resources?

File: content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md
Comment (tiwarisumit3) [inline content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md:29]: 1. Enable the nested cluster option needs to be enabled from the settings tab.

2. Cluster should be launched with this settings.

File: content/docs/04-clusters/4-nested-clusters.md
Comment (TylerGillson) [inline content/docs/04-clusters/4-nested-clusters.md:35]: should be: "with SSL passthrough enabled"

File: content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md
Comment (TylerGillson) [inline content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md:33]: "below are available, when you are enabling" - no comma needed. Should be "below are available when you are enabling"

File: content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md
Comment (TylerGillson) [inline content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md:37]: "an existing cluster and configure it to become a Nested Cluster." --> "an existing cluster and allow it to host Nested Clusters."

File: content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md
Comment (TylerGillson) [inline content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md:39]: "and then, enable the Nested Cluster." --> "and then enable it to host Nested Clusters."

File: content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md
Comment (TylerGillson) [inline content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md:123]: As of last night, quotas are being removed for 2.8

**Review by truegilb — CHANGES_REQUESTED** [review]
Overall looks great - as mentioned please remove quota from the description as it's the latest update that needs to happen in 2.8.

File: content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md
Comment (TylerGillson) [inline content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md:76]: should be "ensure that SSL passthrough is enabled"

**Review by TylerGillson — CHANGES_REQUESTED** [review]

**Review by tiwarisumit3 — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/854#" title="Author self-approved">Siboney1</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/854#pullrequestreview-1092564149" title="Approved">tiwarisumit3</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by tiwarisumit3** [comment]
Apart from minor comments from Tyler to be addressed, overall it looks good to me. Thanks Dorris.

**Review by truegilb — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/854#" title="Author self-approved">Siboney1</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/854#pullrequestreview-1092564149" title="Approved">tiwarisumit3</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/854#pullrequestreview-1093910754" title="Approved">truegilb</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by TylerGillson — APPROVED** [review]

**Pull request merged**
